dags/utils/gcp.py

```diff
@@ -352,7 +352,7 @@ def export_to_parquet(
     connection = GoogleCloudBaseHook(gcp_conn_id=gcp_conn_id)
 
     if destination_table is None:
-        destination_table = table.rsplit(".", 1).pop()
+        destination_table = unqualified_table
```

Comment:
fixes issue in `clients_last_seen_v1_export` and `exact_mau_by_dimensions_export` where the partition qualifier (e.g. `$20191109`) was being included in the s3 path